src/shim/telemetryClient.ts

```diff
@@ -118,12 +120,14 @@ export class TelemetryClient {
      * telemetry bandwidth by aggregating multiple measurements and sending the resulting average at intervals.
      * @param telemetry      Object encapsulating tracking options
      */
-    public trackMetric(telemetry: Contracts.MetricTelemetry): void {
-        // TODO : Create custom metric
-        // let meter = this.client.getMetricHandler().getCustomMetricsHandler().getMeter();
-        // let metricName = "";
-        // let options: MetricOptions = {};
-        // meter.createHistogram(metricName, options)
+    public trackMetric(telemetry: Contracts.MetricPointTelemetry): void {
```

Comment:
Is there any attribute that can be passed for the Metrics in this API?

Reply:
Updated typing and pass any custom properties via the `Histogram.record()` method's `attributes` parameter.